chain/x/zoracle/internal/keeper/keeper.go

```diff
@@ -13,18 +14,91 @@ type Keeper struct {
 	cdc           *codec.Codec
 	CoinKeeper    bank.Keeper
 	StakingKeeper staking.Keeper
+	Paramspace    params.Subspace
```

Comment:
Let's call this `ParamSpace`. I think most of the sdk's packages use that name.

Reply:
done